### PR TITLE
MAINT: adjust covlett for Nature Methods

### PR DESCRIPTION
**Pull request by tylerjereddy**
* adjust the cover letter in preparation
for submission to *Nature Methods* after
receiving Editorial bounce-back from
*Scientific Reports* ("out of scope")

* also included are formatting changes
used for the previous submission cover letter,
so that it fits on a single page,
but phone numbers have again been censored
to protect PII

From the [instructions for *Nature Methods*](https://www.nature.com/nmeth/for-authors/preparing-your-submission), Cover Letter section:

> You must disclose details of any related manuscripts that you have under consideration or in press elsewhere

> You should indicate whether you have had any prior discussions with a Nature Methods editor about the work described in the manuscript.

Those should be addressed now--Editorial Contact is via @stefanv.

Here's the [aims & scope page](https://www.nature.com/nmeth//about/aims). We perhaps need to be cautious of the bio focus, but it is certainly true that we meet "Articles are selected for publication that present broad interest."

### CODE REVIEW

File: scipy-1.0/covlett.tex [hunk 1]
```diff
@@ -5,70 +5,89 @@
...
 
-We have specifically chosen \emph{Scientific Reports} because
-the breadth of acceptable topics matches the variety of algorithms
-our library provides. High-profile manuscripts citing SciPy range
-from the biosciences to recent reports on gravitational waves and
+We have specifically chosen \emph{Nature Methods} because
+SciPy lies at the base of the entire ecosystem of scientific methods
+described in the journal scope, and because SciPy is effectively an
+ever-improving methodological library. High-profile manuscripts citing SciPy 
+extend beyond the biosciences to recent reports on gravitational waves and
```
Comment (mdhaber) [inline scipy-1.0/covlett.tex]: I was going to suggest adding support that SciPy is used in the biosciences. 
"Among the 3000+ articles that cite the SciPy library are hundreds in the field of biology, including several in Nature Methods and other publications by Nature."

Two in Nature Methods:
https://www.nature.com/articles/s41592-018-0114-z
https://www.nature.com/articles/s41592-019-0403-1

Even though not all of the papers in this [quick search](https://scholar.google.com/scholar?hl=en&as_sdt=2005&sciodt=0%2C5&cites=2086009121748039507&scipsc=1&q=biology&btnG=) are actually bioscience-focused, many of them certainly are.

But still, I'm not sure we can hit this criterion:
"To enhance the practical relevance of each paper, description of the method must be accompanied by its validation, its application to an important biological question and results illustrating its performance in comparison to available approaches."

Comment (rgommers) [reply scipy-1.0/covlett.tex]: thanks @mdhaber, those are useful references. from the [NSF proposal](https://figshare.com/articles/Mid-Scale_Research_Infrastructure_-_The_Scientific_Python_Ecosystem/8009441): _Qiime [6] for gene sequencing (13,500 citations)_ is the most high-profile bio tool we could find

> But still, I'm not sure we can hit this criterion:

agreed, but not much we can do about it i'm afraid ....

Comment (mdhaber) [reply scipy-1.0/covlett.tex]: Yeah I figure lets just send it, if we don't have to reformat, and see how it goes.

Comment (tylerjereddy) [reply scipy-1.0/covlett.tex]: I've revised the cover letter based on the feedback above & to reflect the availability of the preprint.

Let me know what you think. Sadly, we run over to a second page now, but maybe that's ok for the extra clarification of relevance. I've somewhat-awkwardly used footnotes for a few citations, but maybe good enough.

Comment (apbard) [reply scipy-1.0/covlett.tex]: Do you think it could help adding those reference and that sentence in the introduction along with the black hole one?

**Comment by mdhaber** [comment]
How about Author/Year only citations, and inline?
```
including several in \emph{Nature Methods} (e.g. \href{
https://www.nature.com/articles/s41592-018-0114-z}
{Gruber \emph{et al.} (2018)}
\href{https://www.nature.com/articles/s41592-019-0403-1}{
; Moen \emph{et al.} (2019)}), one of which
(\href{https://www.nature.com/articles/nmeth.f.303}
{Caporaso \emph{et al.} (2010)}) is cited
```
Since there are links, I think it's OK.

Reduce the margins a tad, and I'll bet it will go on one page.
(I would suggest changes, but I don't really understand how the margins are currently set.)

Also, [the Qiime paper](https://www.nature.com/articles/nmeth.f.303) doesn't list SciPy in the references. I can't access the paper here, unfortunately - can you see if it's mentioned in the text. I know that SciPy was a dependency of the library, but I'm not sure the reader would understand if they were to check the paper.

File: scipy-1.0/covlett.tex [hunk 2]
```diff
@@ -5,70 +5,102 @@
...
-Prof. Syma Khalid,\\
-School of Chemistry,\\
-University of Southampton,\\
-Highfield Campus,\\
-Southampton,\\
-SO17 1BJ, U.K.}
+Dr. Rita Strack,\\
+Nature Methods Editorial Office,\\
+One New York Plaza Suite 4500,\\
+New York, NY 10004,\\
```
Comment (mdhaber) [inline scipy-1.0/covlett.tex]:
```suggestion
New York, NY 10004,
```
For our addresses, USA is on the same line. Save a line to fit on one page.

File: scipy-1.0/covlett.tex [hunk 3]
```diff
@@ -5,70 +5,102 @@
...
-
 \namefrom{Ralf Gommers,
           Matt Haberland,
           and Tyler Reddy}
 \addrfrom{%
     \today\\[10pt]
+    Ralf Gommers,\\
     Quansight Labs,\\
-    The Netherlands\\
+    The Netherlands,\\
```
Comment (mdhaber) [inline scipy-1.0/covlett.tex]: It looks like the commas were added intentionally, but I've never seen that before. Is it specified by the journal or something? [Example cover letters](https://www.google.com/search?rlz=1C1GCEU_enUS820US820&biw=1920&bih=937&tbm=isch&sa=1&ei=fWs7XcjCIZfN-gSpxKbwBg&q=academic+cover+letter+format&oq=academicover+letter+format&gs_l=img.3.0.0i7i30.8188.9275..10149...0.0..0.90.509.7......0....1..gws-wiz-img.......0i8i7i30.YEzf8ENXY08#imgrc=GL-Dc68AJu8TIM:).

**Comment by tylerjereddy** [comment]
Thanks for checking this Matt!

> Also, the Qiime paper doesn't list SciPy in the references. I can't access the paper here, unfortunately - can you see if it's mentioned in the text? I know that SciPy was a dependency of the library, but I'm not sure Dr. Strack would understand if they were to check the paper and see it missing.

Perhaps we should reword to clarify that it is a different kind of dependency--code rather than citation graph since we didn't have a paper! We could also double-check the context in the NSF proposal Ralf mentions. `ctrl+f` searching the PDF didn't yield anything promising for scipy.

File: scipy-1.0/covlett.tex (hunk 3 above)
Comment (tylerjereddy) [reply scipy-1.0/covlett.tex]: I'm happy to surrender in this battle :)

Comment (mdhaber) [reply scipy-1.0/covlett.tex]: Was there a battle before? Or you have an argument for the comas?

**Comment by mdhaber** [comment]
For whatever reason I found it very challenging to just add something that would distinguish between the different kinds of dependencies. One thing led to another, and I found myself with this:

SciPy lies at the base of the ecosystem of life science computational methods. Many studies, including an extremely influential Nature Methods paper (Caporaso et al (2010)), describe software that includes SciPy as a runtime dependency without citation. Others in Nature Methods (e.g. Gruber et al 2018, Moen et al. 2019) -- and some 3000+ other articles -- cite SciPy's website for lack of a better alternative. As SciPy is central to so many biological studies, the research community needs an article about SciPy to reference. Our [preprint](https://arxiv.org/abs/1907.10121) submitted to arXiv only partially fulfills this need; we appreciate your consideration of the manuscript for publication in Nature Methods.

No need to use it. Just wanted to show that I tried.

I'm not

File: scipy-1.0/covlett.tex (hunk 3 above)
Comment (tylerjereddy) [reply scipy-1.0/covlett.tex]: Just a habit of using commas on all but the last line of a contact or address block

File: scipy-1.0/covlett.tex (hunk 1 above)
Comment (tylerjereddy) [reply scipy-1.0/covlett.tex]: @apbard Maybe, though I'm not sure we want to drag on paper adjustments too much more at this stage. I'll mark this discussion as resolved but feel free to open an issue

File: scipy-1.0/covlett.tex (hunk 3 above)
Comment (tylerjereddy) [reply scipy-1.0/covlett.tex]: Ok, I've purged the commas

**Comment by tylerjereddy** [comment]
I think I've followed all of the reviewer suggestions made so far--let me know if anything else is needed.

`make covlett` on this feature branch to preview it

File: scipy-1.0/covlett.tex [hunk 4]
```diff
@@ -5,70 +5,100 @@
...
+    \texttt{mhaberla@calpoly.edu}\\
+    %phone number\\
     \\
-    CCS-7 Applied Computer Science Group,\\
-    Los Alamos National Laboratory,\\
-    Los Alamos, NM, 87545, USA
+    Tyler Reddy\\
+    CCS-7 Applied Computer Science Group\\
+    Los Alamos National Laboratory\\
+    Los Alamos, NM, 87545, USA\\
```
Comment (mdhaber) [inline scipy-1.0/covlett.tex]:
```suggestion
    Los Alamos, NM 87545, USA\\
```
Same for my address. Already correct for Rita.

Comment (mdhaber) [reply scipy-1.0/covlett.tex]: Otherwise LGTM

Comment (tylerjereddy) [reply scipy-1.0/covlett.tex]: ok, done for both addresses

**Comment by mdhaber** [comment]
I think we're set. @rgommers @stefanv?

**Comment by rgommers** [comment]
LGTM

**Comment by tylerjereddy** [comment]
Ok, sounds like we have agreement so I'll put this in / start looking at submitting again soon

**Pull request merged**